docs/src/userguide/language_basics.rst

```diff
@@ -581,11 +837,25 @@ be careful what you do.
 Type Casting
 ------------
 
-Where C uses ``"("`` and ``")"``, Cython uses ``"<"`` and ``">"``. For example::
+Where C uses ``"("`` and ``")"``, Cython uses ``"<"`` and ``">"``. For example
```

Comment:
Pure python text should talk about `cython.cast`

Reply:
done